source/stylesheets/header.scss

```diff
@@ -1,49 +1,71 @@
-details > summary {
-	padding: 1.05em;
-	color: white;
-	font-size: 1.8em;
+$_topic-size: 5rem;
+
+.topics-nav {
+	position: fixed;
+	top: 2rem;
+	left: 2rem;
 }
 
-details {
+.topics-nav__toggle {
+	height: $_topic-size;
+	width: $_topic-size;
+	display: flex;
+	align-items: center;
+	justify-content: center;
 	background: #ed3e44;
+	border-radius: 50%;
 	box-shadow: 0.5px 0.5px 25px 0.5px grey;  
-}
+	z-index: 2;
 
-details[open] {
-	box-shadow: 0.5px 0.5px 25px 0.5px grey;  
 
-}
+	&::-webkit-details-marker {
+		display: none;
+	}
 
-nav {
-	color: white;
-	position: relative;
+	&:hover {
+		cursor: pointer;
+		background-color: #F0AA41;
+	}
+
+	.topics-nav[open] & {
+		background-color: #F0AA41;
+	} 
 }
 
-details ul {
+.topics-nav__menu {
 	padding: 0;
 	margin: 0;
+	position: absolute;
+	top: $_topic-size - 0.5rem;
+	left: $_topic-size / 2;
+	z-index: -1;
+	list-style-type: none;
 }
 
-li {
- display: block;
+.topics-nav__icon {
+	fill: #fff;
+	width: 3rem;
+	height: 3rem;
 }
 
-details li {
+.topics-nav__link {
 	background-color: #4C95C7;
```

Comment:
What do you think about pulling these color values into SASS variables so they're reusable?

Reply:
Working on a separate branch for just variables as we speak, will have a PR for that shortly:
https://trello.com/c/xSMQnSIX/10-define-variables-for-color-typography